source/includes/_tipser-react-elements.md

```diff
@@ -197,58 +199,82 @@ import { Store } from '@tipser/tipser-elements';
+Stand-alone `Checkout` react component displayed on one site;
 
-### Product Card ###
+Modular Checkout component displaying parts of it as different react Components in POS controlled order on different subsites;
 
-The Product Card is an item used for displaying single `Collection` or `Store` component, among others. The font-family used in the description section of the Product Card is inherited from your website's styles, and the font-size is expressed in the relative `em` units controlled in `.te-product-card` class. The default value used there is `12px`, which you can easily change by adding to your CSS the following style:
+### Stand-alone Checkout
+To display `Checkout` as **stand-alone component**, all you need to do is to place it in  your code like any other Tipser Elements React components:
 
-```css
-.te-product-card {
-    font-size: 14px;
-}
+```js
+import { Checkout } from '@tipser/tipser-elements';
+ ...
+<Checkout />
 ```
-
-All the description elements (product name, brand and price) will become bigger / smaller according to the value you specify in the `px` unit. If you wish to change single description element, please use its specific class names: 
-
-`.te-product-card-name` <br/>
-`.te-product-card-brand`<br/>
-`.te-product-card-price`<br/>
-and:<br/>
-`.te-product-card-sales-price`<br/>
-`.te-product-card-price-regular-price`<br/>
-for products on sale.
-
-
-### Cart ###
-
-The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
-
-```css
-.cart-icon {
-    position: fixed;
-    right: 0;
-    top: 121px;
-    background: #fff;
-    padding: 10px;
-    box-shadow: -2px 2px 7px rgba(0,0,0,0.3);
-    z-index: 10;
-}
+The component will be rendered on Your site as shown below:
+[![](checkout_component.png)](/images/checkout_component.png)
+ 
+ 
+## Modular Checkout
+Use this more advanced way of displaying Checkout, if you want to have more control over its parts.
+
+The `Checkout` consists of several react components:
+
+```html
+<Checkout>
+  <CartProducts />
+  <CustomerAddressDelivery />
+  <CartPromoCode />
+  <CartSummary />
+  <CheckoutPayment />
+  <CustomerAddressBilling />
+</Checkout>
 ```
 
-### Adding Primary color ###
-
-If you'd like to unify our design with your own color-theme, you can use our primary-color [configuration option](#primary-color). If your primary color is a bright one, you might want to change also the text color of the elements with primary color background. For example in buttons by default the color we use is white.
-
-```css
+To make them all connected, you need to place them in a wrapper component `<Checkout/>` and pass all the data via props `{...checkout}`:
```

Comment:
To make them all connected

maybe

to connect them together

or 

to wire them up together

?